fog/view/enclave/api/src/lib.rs

```diff
@@ -187,6 +199,8 @@ pub enum Error {
     Poison,
     /// Enclave not initialized
     EnclaveNotInitialized,
+    /// Cipher encryption failed
+    Cipher,
```

Comment:
It seems useful to include the actual `CipherError`

```suggestion
    /// Cipher encryption failed: {0}
    Cipher(CipherError),
```
(also `use mc_crypto_noise::CipherError`)

Reply:
Done.